pwiz_tools/Skyline/Controls/Databinding/DataboundGridControl.cs

```diff
@@ -964,13 +837,12 @@ private void pCAToolStripMenuItem_Click(object sender, EventArgs e)
 
         public void ShowPcaPlot()
         {
-            var resultsTuple = GetClusteredResults();
-            var pcaPlot = new PcaPlot()
+            var pcaPlot = new PcaPlot

```

Comment:
Same as above. Do these get preserved in the .sky.view and restored, and how do we ensure that we only get one in a specific context? i.e. If I execute exactly the same command to show this plot 5 times, will I get 5 of the same graphs? Do they now update properly based on a document change? If so, it seems undesirable to be making new copies.